modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2693,6 +2693,56 @@ describe('V2 Wallet:', function () {
         });
       });
 
+      it.only('should build an enable token transaction for cold wallets', async function () {
+        const recipients = [];
+        const tokenName = 'tcoin:tokenName';
+        const prebuildTxWithIntent = sandbox.stub(TssUtils.prototype, 'prebuildTxWithIntent');
+        txRequest.walletType = 'cold';
+        prebuildTxWithIntent.resolves(txRequest);
+        // TODO(BG-59686): this is not doing anything if we don't check the return value, we should also move this check to happen after we invoke prebuildTransaction
```

Comment:
Please use a valid ticket, the BG project has been deprecated for a while now...